en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -131,6 +132,22 @@ To repair a connection:
 . Click the change icon in the trigger or action that requires repair, and then click *Change Connection*.
 . Click *Repair*.
 
+== Delete a Connection
+
+If you no longer need a specific OAuth connection, you can delete it.
```

Comment:
This is actually now for all connections, not just OAuth. (So you can just remove the word OAuth)

Reply:
+1